settings/mp_defaults.json

```
@@ -42,5 +42,6 @@
   "skipModSecurityWarning"      : [ "local", false ],
   "highlightQueuedPlayers"      : [ "local", true  ],
   "playerlistLeftclick"         : [ "cloud", 0 ],
+  "enableNewChatDesign"         : [ "cloud", true ],
```

Because it's a relatively new option I'd like to first get some community feedback before making it the default.
```suggestion
  "useUiAppRedesign"            : [ "cloud", false ],
```